compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -865,7 +865,10 @@ execDocTest opts files =
           -- This is horrible but we do not have a way to change the import paths in a running
           -- IdeState at the moment.
           pure $ nubOrd $ mapMaybe (uncurry moduleImportPath) (zip files' pmS)
-      opts <- pure opts { optImportPath = importPaths <> optImportPath opts, optHaddock = Haddock True }
+      opts <- pure opts
+        { optImportPath = importPaths <> optImportPath opts
+        , optHaddock = Haddock True
+        , optEnableScenarios = EnableScenarios True }
```

Comment:
This is a slightly different change: for doctests, we now enable scenarios implicitly since that's an implementation detail. Eventually we might want to change the implementation of `daml doctest` to use `Script` instead of `Scenario`, but that would mean requiring packages with doctests to depend on `daml-script` even if they don't define any scripts.